### PR TITLE
Debug resources page

### PR DESCRIPTION
**Pull request by YamanSammani**
In this Pr, I fixed the mouse hover and fixed the resources translation.

### CODE REVIEW

**Review by ibrahimaljarad — APPROVED** [review]

File: src/containers/ResourcesPage/index.jsx [hunk 1]
```diff
@@ -6,39 +6,34 @@ import resourcesForEducation from "../../images/resourcesForEducation.png";
...
-  },
-];
-
 const ResourcesPage = () => {
   const [t] = useTranslation();
-
+  const initResources = [
+    {
+      id: 1,
+      title: t("resourcesPage.title1"),
```
Comment (louisrli) [inline src/containers/ResourcesPage/index.jsx]: use an array, almost never call something title1 title2

Comment (louisrli) [reply src/containers/ResourcesPage/index.jsx]: an array in the translations. work with tarkan if needed

Comment (YamanSammani) [reply src/containers/ResourcesPage/index.jsx]: @louisrli , for the naming, I will update them. but regarding the array, could you kindly elaborate more. I don't see the point of using an array in here.

Comment (YamanSammani) [reply src/containers/ResourcesPage/index.jsx]: PTAL

File: src/containers/ResourcesPage/index.jsx [hunk 2]
```diff
@@ -6,39 +6,34 @@ import resourcesForEducation from "../../images/resourcesForEducation.png";
...
-  },
-];
-
 const ResourcesPage = () => {
   const [t] = useTranslation();
-
+  const initResources = [
+    {
+      id: 1,
+      title: t("resourcesPage.resourceOneTitle"),
```
Comment (louisrli) [inline src/containers/ResourcesPage/index.jsx]: resourceOne is not better than title1

they also have the same structure, thats why an array makes sense, either that or meaningful names

your two options are to make the code either look like this:

t("resourcesPage.resources[0].title")
t("resourcesPage.resources[0].text")
(or something like this, i cant remember the syntax for arrays)

or like this:
t("resourcesPage.resources.climateChangeHandbook.text") (or some other meaningful name)

both of these are cleaner, the second provides more meaning if the descriptions actually have to do with the resource themselves. which it does.

imagine your teammate needs to change this code, and they need to change the order of the arrays. and they need to decide ,"hey should i use resourceOneText or resourceTwoText for climate change handbook.pdf?" 

* more readable: t("resourcesPage.resources.climateChangeHandbook.title")
* not readable: (".resourceOneText")

Comment (YamanSammani) [reply src/containers/ResourcesPage/index.jsx]: in this way, it's much much much clearer :D, Thanks. PTAL.

File: src/components/Resource/index.css [hunk 3]
```diff
@@ -45,3 +47,7 @@
   padding: 6px 10px;
   border-radius: 7px;
 }
+.resource__body__content__footer button:hover {
```
Comment (louisrli) [inline src/components/Resource/index.css:51]: newline above

File: src/containers/ResourcesPage/__snapshots__/index.test.js.snap [hunk 4]
```diff
@@ -111,12 +111,12 @@ exports[`ResourcesPage renders correctly 1`] = `
...
           className="resource__body__content__title"
         >
-          Who Causes Climate changes - TROCAIRE
+          resourcesPage.resource.whoCausesClimateChangesHandbook.title
         </div>
         <div
           className="resource__body__content__description"
         >
-          Children explore who contribute to climate change by looking at activities related to climate change using climate change bingo sheet, creating clouds that list activates causing gas emissions and represent the world population by dividing themself.
+          resourcesPage.resource.whoCausesClimateChangesHandbook.titleext
```
Comment (louisrli) [inline src/containers/ResourcesPage/__snapshots__/index.test.js.snap]: titleext

File: src/containers/ResourcesPage/index.jsx [hunk 5]
```diff
@@ -6,39 +6,38 @@ import resourcesForEducation from "../../images/resourcesForEducation.png";
...
+      body: t("resourcesPage.resources.educationOnClimateChangeHandbook.text"),
+      pic: resourcesForEducation,
+      url:
+        "gs://capstone-istanbul-juniper.appspot.com/ResourcesPage/Resources list_Climate Change.pdf",
+    },
+    {
+      id: 3,
+      title: t("resourcesPage.resource.whoCausesClimateChangesHandbook.title"),
+      body: t(
+        "resourcesPage.resource.whoCausesClimateChangesHandbook.titleext"
```
Comment (louisrli) [inline src/containers/ResourcesPage/index.jsx]: titleext

Comment (YamanSammani) [reply src/containers/ResourcesPage/index.jsx]: I have a PhD in Typo :D, PTAL.

File: src/components/Resource/index.css (hunk 3 above)
Comment (YamanSammani) [reply src/components/Resource/index.css:51]: PTAL

**Comment by louisrli** [comment]
you should also proofread your work :)

File: src/locales/en.json [hunk 6]
```diff
@@ -277,8 +277,22 @@
...
+        "title": "Our Climate Change",
+        "text": "This book is a child friendly handbook meant to explain the concepts of climate change in an easily understandable manner and language. It aims to link behavioural choices to the changes observed in our climate."
+      },
+      "educationOnClimateChangeHandbook": {
+        "title": "Resources for Education on Climate Change - UNESCO",
+        "text": "This publication focuses on the challenges, opportunities and good practices of climate change."
+      },
+      "whoCausesClimateChangesHandbook": {
+        "title": "Who Causes Climate changes - TROCAIRE",
+        "test": "Children explore who contribute to climate change by looking at activities related to climate change using climate change bingo sheet"
```
Comment (louisrli) [inline src/locales/en.json]: fix, and please test your code and translations

Comment (louisrli) [reply src/locales/en.json]: and proofread everything before sending it again

**Comment by YamanSammani** [comment]
@louisrli , PTAL. I hope it's clean now 😅.

**Review by louisrli — APPROVED** [review]

**Pull request merged**